app/controllers/trainees/subject_specialisms_controller.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+module Trainees
+  class SubjectSpecialismsController < ApplicationController
```

Comment:
This controller is currently only handing language subjects, but the name doesn't reflect that.

Will need discussion with https://trello.com/c/uMJ3U4dm/2102-l-subjects-allow-users-to-choose-un-mappable-subjects-general

Reply:
I think if we change this to `LanguageSpecialismsController` it will avoid a conflict with https://github.com/DFE-Digital/register-trainee-teachers/pull/1093